model/src/main/java/uk/gov/hmcts/payment/api/service/PciPalPaymentService.java

```diff
@@ -74,6 +72,27 @@ else if (serviceType.equalsIgnoreCase(SERVICE_TYPE_PROBATE))
         });
     }
 
+    private String getppAccountId(String serviceType) {
+        String ppAccountID = null;
+        if (serviceType.equalsIgnoreCase(SERVICE_TYPE_DIVORCE)) {
+            ppAccountID = ppAccountIDDivorce;
+        }
+        else if (serviceType.equalsIgnoreCase(SERVICE_TYPE_CMC)) {
+            ppAccountID = ppAccountIDCmc;
+        }
+        else if (serviceType.equalsIgnoreCase(SERVICE_TYPE_PROBATE)) {
+            ppAccountID = ppAccountIDProbate;
+        }
+        else if (serviceType.equalsIgnoreCase(SERVICE_TYPE_FINREM)) {
+            ppAccountID = ppAccountIDFinrem;
```

Comment:
Since we are on-boarding many Services for telephony, this list will grow, maybe if we have a static map of Services to the ppAccountIDs and retrieve by the service, it would look good. if we don't have time now then we can at next service on-boarding.

Reply:
@AyanDuttaHMCTS - Yeah we will definitely take this point in the next on-boarding.

Reply:
@AyanDuttaHMCTS - Yeah we will definitely take this point in the next on-boarding.